### PR TITLE
connect-to-compute-cluster tutorial and slides from dagobah

### PR DESCRIPTION
**Pull request by bgruening**
ping @Slugger70 @natefoo

### CODE REVIEW

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+layout: tutorial_slides
+topic_name: admin
+tutorial_name: connect-to-compute-cluster
+logo: "GTN"
+---
+layout: true
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slides.html]: Why is there a duplication in the metadata here?

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+---
+# Exercise
+
+Explore different ways to route jobs to different compute resources
+
+[Advanced Galaxy Job Configurations](./tutorial.html)
+
+---
+
+layout: true
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slides.html]: Are these metadata needed too?

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html (hunk 1 above)
Comment (bebatut) [reply topics/admin/tutorials/connect-to-compute-cluster/slides.html]: I understood why. I will revert my changes in other PR. But we should fix also the global layout if you think it should be like this

File: topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html [hunk 3]
```diff
@@ -0,0 +1,842 @@
...
+
+<H2>Authentication and Security</H2>
+Define the method used for authenticating communicating between SLURM components.<BR>
+Select one value for <B>AuthType</B>:<BR>
+<input type="radio" name="auth_type" value="none"> <B>None</B>: No authentication,
+not recommended production use<br>
+<input type="radio" name="auth_type" value="authd"> <B>Authd</B>: Brent Chun's
+<A href="http://www.theether.org/authd/">authd</A><BR>
+<input type="radio" name="auth_type" value="munge" checked> <B>Munge</B>: LLNL's
+<A href="http://home.gna.org/munge/">Munge</A><BR>
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html]: This link is failing (time out)

File: topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html [hunk 4]
```diff
@@ -0,0 +1,842 @@
...
+<input type="radio" name="proctrack_type" value="pgid" checked> <B>Pgid</B>: Use Unix
+Process Group ID, processes changing their process group ID can escape from SLURM
+control<BR>
+<input type="radio" name="proctrack_type" value="linuxproc"> <B>LinuxProc</B>: Use
+parent process ID records, required for MPICH-GM use, processes can escape
+from SLURM control<BR>
+<input type="radio" name="proctrack_type" value="rms"> <B>RMS</B>: Use Quadrics
+kernel infrastructure, recommended for systems where this is available<BR>
+<input type="radio" name="proctrack_type" value="sgi_job"> <B>SGI's PAGG
+module</B>: Use <A HREF="http://oss.sgi.com/projects/pagg/">SGI's Process
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html]: This link is failing (404 No error)

File: topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html [hunk 5]
```diff
@@ -0,0 +1,842 @@
...
+<P>
+
+<BR>
+<BR>
+<input type=button value="Submit" onClick="javascript:displayfile()">
+<input type=reset value="Reset Form">
+<P>
+</FORM>
+<HR>
+<a href="disclaimer.html" target="_blank" class="privacy">Legal Notices</a><br>
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html]: `disclaimer.html` does not exists there

File: topics/admin/tutorials/connect-to-compute-cluster/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,855 @@
...
+- The various ways in which tools can be mapped to destinations, both statically and dynamically
+- How to write a dynamic tool destination (DTD)
+- How to write a dynamic python function destination
+- How to use the job resource parameter selection feature
+
+### Further Reading
+
+- The [sample dynamic tool destination config file](https://github.com/galaxyproject/galaxy/blob/dev/config/tool_destinations.yml.sample) fully describes the configuration language
+- [Dynamic destination documentation](https://wiki.galaxyproject.org/Admin/Config/Jobs#Dynamic_Destination_Mapping)
+- Job resource parameters are not as well documented as they could be, but the [sample configuration file](https://github.com/galaxyproject/usegalaxy-playbook/blob/master/files/galaxy/test.galaxyproject.org/config/job_resource_params_conf.xml) shows some of the possibilities.
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/tutorial.md]: `https://github.com/galaxyproject/usegalaxy-playbook/blob/master/files/galaxy/test.galaxyproject.org/config/job_resource_params_conf.xml` does not exist

File: topics/admin/tutorials/connect-to-compute-cluster/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,855 @@
...
+- [usegalaxy.org's job_conf.xml](https://github.com/galaxyproject/usegalaxy-playbook/blob/master/templates/galaxy/usegalaxy.org/config/job_conf.xml.j2) is publicly available for reference.
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/tutorial.md]: `https://github.com/galaxyproject/usegalaxy-playbook/blob/master/templates/galaxy/usegalaxy.org/config/job_conf.xml.j2` does not exist

File: topics/admin/tutorials/connect-to-compute-cluster/slides.html [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+---
+# Cluster options
+
+- Slurm
+- Condor
+- Torque
+- PBS Pro
+- LSF
+- SGE derivatives maybe?
+- Any other [DRMAA](https://www.drmaa.org/)-supported DRM
```
Comment (bebatut) [inline topics/admin/tutorials/connect-to-compute-cluster/slides.html]: `https://www.drmaa.org/ failed` got `response code 0`

Comment (bgruening) [reply topics/admin/tutorials/connect-to-compute-cluster/slides.html]: but it is working

File: topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html (hunk 3 above)
Comment (bgruening) [reply topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html]: It seems to be the correct URL. I can change it to https://github.com/dun/munge in the meantime.

File: topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html (hunk 4 above)
Comment (bgruening) [reply topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html]: SGI is not anymore, removed the link.

File: topics/admin/tutorials/connect-to-compute-cluster/tutorial.md (hunk 7 above)
Comment (bgruening) [reply topics/admin/tutorials/connect-to-compute-cluster/tutorial.md]: fixed

File: topics/admin/tutorials/connect-to-compute-cluster/tutorial.md (hunk 6 above)
Comment (bgruening) [reply topics/admin/tutorials/connect-to-compute-cluster/tutorial.md]: fixed

**Pull request merged**